src/lib/turtle/__init__.js

```diff
@@ -1017,28 +1017,6 @@ function generateTurtleModule(_target) {
         };
         proto.$shape.minArgs = 0;
 
-        proto.$window_width = function() {
-            return this._screen.$window_width();
-        };
-
-        proto.$window_height = function() {
-            return this._screen.$window_height();
-        };
-
-        proto.$tracer = function(n, delay) {
```

Comment:
@bzwheeler, This appears to have the unintended consequence of removing the tracer method from turtle.  Even though I've been running this locally with the latest build I didn't notice this until I moved this latest set of changes to the live site!  Ugh!   t.tracer(10,30)  Now throws an error that tracer is not a method of turtle.